src/main/StartUp.java

```diff
@@ -12,8 +14,10 @@
 import java.time.Instant;
 
 public class StartUp {
```

Comment:
I don't think there is a need for a constructor in this class. The program can start from **main(String[] args)** method.

Reply:
You're saying to call the `prepareGUI() `and `runEvaluation()` methods directly from `main`?

Reply:
Yes.